### PR TITLE
Added colour command

### PR DESCRIPTION
**Pull request by Cyan101**

### CODE REVIEW

**Comment by Cyan101** [comment]
should also probably rename the file to responses.rb, i'd do this myself but im not sure if this file is referenced anywhere else in the code

**Comment by z64** [comment]
@Cyan101 the files are lazily included and not bound by filename ; it uses my modular bot template

**Comment by Cyan101** [comment]
Renamed the file for simplicity and so it matches the database

File: lib/modules/commands/responses.rb [hunk 1]
```diff
@@ -25,6 +25,16 @@ module Actions
         event << "#{event.user.display_name} #{response} boops #{thing}"
         event.message.delete
       end
+      
+      command(:colour,
+             description: 'Returns matching colours',
+             usage: "#{BOT.prefix} colour") do |event|
+        thing = thing.join(' ')
```
Comment (z64) [inline lib/modules/commands/responses.rb:32]: This does not exist, nor is it used for anything

File: lib/modules/commands/responses.rb [hunk 2]
```diff
@@ -25,6 +25,16 @@ module Actions
...
+        response = Database::Response.where(key: 'colour')
+                                     .all.sample.response
+        event << "#{event.user.display_name} #{response}"
+        event.message.delete
```
Comment (z64) [inline lib/modules/commands/responses.rb:36]: I would prefer for us not to delete invoking messages for things like this where data is returned. For actions it makes sense, for this, it doesn't.

**Review by z64 — CHANGES_REQUESTED** [review]
Please add the `:color` command as well.

**Pull request closed without merging**